golangci.yaml

```diff
@@ -0,0 +1,7 @@
+linters:
```

Comment:
Why id this file here if we do not us golangci

Reply:
I've added golangci-lint as a new Makefile target. We can fix the lint issues in a separate PR and enable the linter for our CI.